src/Stratis.Bitcoin.Features.SignalR/Broadcasters/WalletInfoBroadcaster.cs

```diff
@@ -1,111 +1,103 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.Extensions.Logging;
-using NBitcoin;
-using Stratis.Bitcoin.AsyncWork;
-using Stratis.Bitcoin.Connection;
-using Stratis.Bitcoin.Features.SignalR.Events;
-using Stratis.Bitcoin.Features.Wallet;
-using Stratis.Bitcoin.Features.Wallet.Interfaces;
-using Stratis.Bitcoin.Utilities;
-using Stratis.Bitcoin.Features.Wallet.Models;
-using Stratis.Bitcoin.Consensus;
-
 namespace Stratis.Bitcoin.Features.SignalR.Broadcasters
 {
+    using System;
```

Comment:
Move back to outside of namespace declaration.

Reply:
Done